client/src/components/Intro/intro.css

```
@@ -24,6 +24,10 @@
   border: none;
 }
 
+.row {
+  margin:0;
+}
+
```

```suggestion
.row {
  margin: 0;
}

```

Should resolve the linting issue.

---

> Should resolve the linting issue.
Hi @nhcarrigan 
Done the changes 
ThankYou